shared/chat/conversation/header-area/container.tsx

```diff
@@ -1,37 +0,0 @@
-import * as React from 'react'
```

Comment:
this file basically did nothing, so hoisting the child up a level and replacing it